config/task.go

```diff
@@ -281,6 +281,8 @@ func (c *TaskConfig) Validate() error {
 					"task.condition.regexp or at least one service in " +
 					"task.services to be configured")
 			}
+		case *ConsulKVConditionConfig:
+			return fmt.Errorf("at least one service must be configured")
```

Comment:
Could you add more context to this error -- like including that a task with the KV condition requires at least one service to be configured?

Reply:
Added!
```
2021/09/01 15:50:16 [ERR] (cli) error validating configuration: consul-kv condition requires at least one service to be configured in task.services
```